subcommittees/policy-CoC/README.md

```diff
@@ -0,0 +1,33 @@
+# The Code of Conduct Policy Subcommittee
+
+## Purpose and Scope
+
+Enforce the Code of Conduct (CoC) in all parts of the Carpentries community
+
+## What We Do
+
+* communicate with the Carpentries community about the CoC, as well as reporting and enforcement guidelines
+* review and respond to reports of CoC violations as deemed appropriate by the situation
+
+## Contact
+
+* policy@carpentries.org
+* confidential@carpentries.org
+
+## Current Members
+
+* Pauline Barmby - chair (community)
+* Chris Hamm (community)
+* Simon Waldman (community)
+* Erin Becker (staff)
+* Jonah Duckles (staff)
+
+## Past Members
+
+We don't have any past members yet!  When current members leave the
+Subcommittee we will move their entries into this section.
+
+[blog]: https://software-carpentry.org/blog/
+[blog-archives]: https://software-carpentry.org/blog/categories/#SLUG
+[mailing-list]: http://lists.software-carpentry.org/listinfo/SLUG
+[mailing-list-archives]: http://lists.software-carpentry.org/pipermail/SLUG/
```

Comment:
You've removed the text which consumes these references so you should either:

* Drop the references.
* Add/restore some text consuming them.

I'd like to see blog links back, or some other words about how the goal of “communicate with the Carpentries community about the CoC” will be carried out.  If you're a member of the Carpentries community interested in receiving that communication, where do you look?